etc/emqx.conf

```diff
@@ -2264,13 +2264,26 @@ broker.session_locking_strategy = quorum
 ## Dispatch strategy for shared subscription
 ##
 ## Value: Enum
+## - hash_clientid
+## - hash # same as hash_clientid
+## - hash_topic
+## - local
 ## - random
 ## - round_robin
 ## - sticky
-## - hash # same as hash_clientid
+broker.shared_subscription_strategy = random
+
+## Per group dispatch strategy for shared subscription
+##
+## Value: Enum
 ## - hash_clientid
+## - hash # same as hash_clientid
 ## - hash_topic
-broker.shared_subscription_strategy = random
+## - local
+## - random
+## - round_robin
+## - sticky
+broker.sample_group.shared_subscription_strategy = local
```

Comment:
Is it best to comment out this line?
For the default confs, we should not configure a different shared subscription stragegy for any group